opensrp-immunization/src/main/java/org/smartregister/immunization/repository/VaccineRepository.java

```diff
@@ -193,12 +194,12 @@ public static String addHyphen(String s) {
         return s;
     }
 
-    public List<Vaccine> findUnSyncedBeforeTime(int hours) {
+    public List<Vaccine> findUnSyncedBeforeTime(int minutes) {
```

Comment:
These is is a public method, should we perhaps add a new method for  backward compatibility? e.g _findSyncedBeforeTimeMinutes(minutes)_ 

Otherwise this breaks for any implementation that doesn't update their build configs **VACCINE_SYNC_TIME** to be in minutes

Reply:
Nice catch. Yea, it's a public method but no one actually uses this method https://github.com/search?p=1&q=org%3AOpenSRP+findUnSyncedBeforeTime&type=Code

I have changed it so that the **BuildConfig** does not need to be changed. The time is retrieved from the `ImmunizationLibrary` class. In case, the current default does not work. The test in the `ImmunizationLibraryTest` will fail